app/scripts/superuser.py

```diff
@@ -0,0 +1,35 @@
+import argparse
+from app.crud.user import CRUDUser
```

Comment:
YOu need to use CRUDUser instance instead of class, crud_user

Reply:
@vladysllav fixed